src/System.Text.Json/tests/Utf8JsonReaderTests.cs

```diff
@@ -4364,7 +4364,8 @@ public static IEnumerable<object[]> InvalidJsonStrings
                     new object[] {"fals", 0, 4},
                     new object[] {"\"a\u6F22\u5B57ge\":", 0, 11},
                     new object[] {"{\"a\u6F22\u5B57ge\":", 0, 13},
-                    new object[] {"{\"name\":\"A\u6F22\u5B57hso", 0, 8},
+                    new object[] {"{\"name\":\"A\u6F22\u5B57hso", 0, 18},
```

Comment:
Previously, we would report the position of the start of the JSON string (that doesn't have an end quote within the payload), as part of the exception. Now, we are reporting the end of the payload.